frontend/next.config.mjs

```diff
@@ -1,12 +1,31 @@
-import dotenv from 'dotenv';
+
+import dotenv from "dotenv";
+
 
 dotenv.config({ path: `../.env` });
 
 /** @type {import('next').NextConfig} */
 const nextConfig = {
     env: {
-      BACKEND_HOST: process.env.BACKEND_HOST,
-      BACKEND_PORT: process.env.BACKEND_PORT
+
+        BACKEND_HOST: process.env.BACKEND_HOST,
+        BACKEND_PORT: process.env.BACKEND_PORT,
+        GOOGLE_CLIENT_ID: process.env.GOOGLE_CLIENT_ID,
+        GOOGLE_CLIENT_SECRET: process.env.GOOGLE_CLIENT_SECRET,
+        GOOGLE_CALLBACK_URL: process.env.GOOGLE_CALLBACK_URL,
```

Comment:
Those envs are not passed by docker compose. `docker-compose.yml` should be updated.